Sources/SwiftParserDiagnostics/ParseDiagnosticsGenerator.swift

```diff
@@ -279,9 +279,9 @@ public class ParseDiagnosticsGenerator: SyntaxAnyVisitor {
       }
     }
 
-    if let throwsSpecifier = node.throwsSpecifier {
+    if let throwsSpecifier = node.throwsSpecifier, node.thrownType == nil {
```

Comment:
Why do we need to check that `thrownType` is `nil` here? We should be able to handle `throws(MyError) async` as well.

Reply:
We don't need to do the check